pkg/controller/restore_controller.go

```diff
@@ -361,9 +398,10 @@ func (c *restoreController) completeAndValidate(restore *api.Restore) []string {
 		}
 	}
 
-	// validate that we can fetch the source backup
-	if _, err := c.fetchBackup(c.bucket, restore.Spec.BackupName); err != nil {
-		return append(validationErrors, fmt.Sprintf("Error retrieving backup: %v", err))
+	// re-use the validation error we got when doing an earlier lookup to check for
+	// a given backup's schedule name for metrics.
+	if fetchErr != nil {
```

Comment:
In the current state of the code, `fetchErr` will **always** be nil if the restore was created with a non-nil `.spec.scheduleName`, essentially bypassing this validation.  I have some potential thoughts on how to simplify the whole flow around fetch/fetchErr but need to understand the answer to my q around "better tracking" first.